app/models/accredited_individual.rb

```diff
@@ -41,16 +43,35 @@ class AccreditedIndividual < ApplicationRecord
   # @return [AccreditedIndividual::ActiveRecord_Relation] an ActiveRecord_Relation of
   #   all individuals matching the search criteria
   def self.find_within_max_distance(long, lat, max_distance = AccreditedRepresentation::Constants::DEFAULT_MAX_DISTANCE)
-    query = 'ST_DWithin(ST_SetSRID(ST_MakePoint(:long, :lat), 4326)::geography, location, :max_distance)'
+    query = 'ST_DWithin(ST_SetSRID(ST_MakePoint(:long, :lat), 4326)::geography,' \
+            'accredited_individuals.location, :max_distance)'
     params = { long:, lat:, max_distance: }
 
     where(query, params)
   end
 
+  #
+  # Find all [AccreditedIndividuals] with a full name with at least the threshold value of
+  #   word similarity. This gives us a way to fuzzy search for names.
+  # @param search_phrase [String] the word, words, or phrase we want individuals with full names similar to
+  # @param threshold [Float] the strictness of word matching between 0 and 1. Values closer to zero allow for words to
+  #   be less similar.
+  #
+  # @return [AccreditedIndividual::ActiveRecord_Relation] an ActiveRecord_Relation of
+  #   all individuals matching the search criteria
+  def self.find_with_full_name_similar_to(search_phrase, threshold = DEFAULT_FUZZY_THRESHOLD)
+    where('word_similarity(?, accredited_individuals.full_name) >= ?', search_phrase, threshold)
+  end
+
   # return all poa_codes associated with the individual
   #
   # @return [Array<String>]
   def poa_codes
     ([poa_code] + accredited_organizations.pluck(:poa_code)).compact
   end
+
+  # This method needs to exist on the model so [Common::Collection] doesn't blow up when trying to paginate
+  def self.max_per_page
+    100
```

Comment:
I don't think you're referencing `Veteran::Service::Constants::MAX_PER_PAGE` anywhere. Should you replace `100` with the reference?

Reply:
That constant lives in the `veteran` module and we don't want to reference it. I can add a new constant to `AccreditedRepresentation::Constants` to reference if needed. The controller will never use this number as long as it's > 10 and we wouldn't even need this method if the collection pagination method used a `try`.

Reply:
Haha - yes, we don't want to reference the `Veteran::Service::Constants` - I didn't realize that class is in the other module.

Reply:
Added the constant